nautobot/docs/development/apps/index.md

```diff
@@ -25,28 +22,13 @@ The Nautobot App architecture allows for Apps to do any or all of the following:
 * **Add home page content.** An App can add custom items or custom panels to the Nautobot home page.
 * **Add content to existing model detail views.** An App can inject custom HTML content within the view of a core Nautobot model. This content can appear in the left column, right column, or full width of the page, and can also include custom buttons at the top of the page.
 
-+++ 1.2.0
-    * **Add a banner.** An App can add a custom banner to the top of any appropriate views.
-
-+++ 1.4.0
-    * **Add extra tabs to existing model detail views.** An App can inject additional tabs which will appear at the end of the object detail tabs list.
```

Comment:
These items should be rolled into the above bullet list rather than removed entirely.

Reply:
Sounds good. I am adding back in my updates
```
* **Add a banner.** An App can add a custom banner to the top of any appropriate views.
* **Add extra tabs to existing model detail views.** An App can inject additional tabs which will appear at the end of the object detail tabs list.
```
but keeping 
```
+/- 1.5.2
    The `nautobot.apps` namespace was added in Nautobot 1.5.2. If developing apps to be backwards-compatible with older versions of Nautobot, please refer to the app developer documentation of your required Nautobot version.
```
removed.